webapp/sources/pom.xml

```diff
@@ -167,14 +167,7 @@ limitations under the License.
               -Xlint:_,-nullary-unit,-missing-interpolator -Yno-adapted-args -Ywarn-dead-code -Ywarn-extra-implicit -Ywarn-inaccessible
               -Ywarn-infer-any -Ywarn-nullary-override -Ywarn-numeric-widen -Ywarn-unused:imports -Ywarn-unused:locals -Ywarn-unused:privates
             -->
-            <arg>-deprecation</arg>                  <!-- Emit warning and location for usages of deprecated APIs. -->
-            <arg>-explaintypes</arg>                 <!-- Explain type errors in more detail. -->
-            <arg>-feature</arg>                      <!-- Emit warning and location for usages of features that should be imported explicitly. -->
-            <arg>-unchecked</arg>                    <!-- Enable additional warnings where generated code depends on assumptions. -->
-            <arg>-language:existentials</arg>        <!-- Existential types (besides wildcard types) can be written and inferred -->
-            <arg>-language:higherKinds</arg>         <!-- Allow higher-kinded types -->
-            <arg>-language:implicitConversions</arg> <!-- Allow definition of implicit functions called views -->
-            <arg>-Xcheckinit</arg>                   <!--  Wrap field accessors to throw an exception on uninitialized access. -->
```

Comment:
Why removing all these flags ?

Reply:
I just had an issue with my IDE which auto-formats the pom.xml, and I didn't check the diff ^^